core/domain/exp_domain.py

```diff
@@ -617,6 +619,8 @@ def __init__(
         self.last_updated = last_updated
         self.auto_tts_enabled = auto_tts_enabled
         self.correctness_feedback_enabled = correctness_feedback_enabled
+        self.android_proto_size_is_stale = True
```

Comment:
You can probably make this private too, actually. Is it referenced outside the class?

Maybe it should also be self._cached_android_proto_size_is_stale, instead.

Reply:
Fixed